src/Actions/Product/CreateProduct.php

```diff
@@ -54,6 +56,13 @@ public function performAction(): Product
             $product->prices()->createMany($prices);
         }
 
+        if ($productCrossSellings) {
```

Comment:
if statement unnecessary, foreach is enough because its defaulted to empty array. Its only an issue if you try to loop through something that isn't countable.